cadence/nfts/blind-edition-nft/transactions/deploy.cdc

```diff
@@ -1,10 +1,44 @@
+import MetadataViews from {{{ imports.MetadataViews }}}
+import FungibleToken from {{{ imports.FungibleToken }}}
+
+pub fun prepareRoyalties(
+    addresses: [Address],
+    receiverPaths: [PublicPath],
+    cuts: [UFix64],
+    descriptions: [String]
+): [MetadataViews.Royalty] {
+    let royalties: [MetadataViews.Royalty] = []
+
+    for i, address in addresses {
```

Comment:
Do we need any nullable check on `addresses` in case we don't use royalties? Or would cadence be able to handle this case?

Reply:
To omit royalties you just need to pass an empty array to this transaction. The Node.js library automatically converts a null value to an empty array. Test case here: https://github.com/packagelabs/freshmint/pull/138/files#diff-22d93c09bfe9d402b5b04612fee6e10f070e8889cc7e11bdbc6a89b8eb57ad33R45